operations/helm/charts/mimir-distributed/RELEASE.md

```diff
@@ -19,80 +19,142 @@ Weekly releases have the version `x.y.z-weekly.w`, for example `3.1.0-weekly.196
 
-## Release process for a release candidate or final release
-
-The following steps apply to a release candidate or a final release.
+## Release process for a release candidate
 
 1. Determine the Helm chart version number.
 
    [The Chart.yaml file](https://helm.sh/docs/topics/charts/#the-chartyaml-file) requires semantic versioning:
 
    - Release candidates have the version `x.y.z-rc.w`, for example `3.1.0-rc.7`.
-   - The final version has the version `x.y.z`, for example `3.1.0`.
 
    > **Note**: You must precede the release candidate number (such as `7`) with a dot (`.`).
 
-1. If you do not have a release branch, create one:
+1. Prepare changelog.
 
-   a. Create a PR, whose target is `main`, that updates the [Helm chart changelog](https://github.com/grafana/mimir/blob/main/operations/helm/charts/mimir-distributed/CHANGELOG.md), and move any `## main / unreleased` items under this release’s version.
+   - Create a PR, whose target is `main`, that updates the [Helm chart changelog](https://github.com/grafana/mimir/blob/main/operations/helm/charts/mimir-distributed/CHANGELOG.md), and move any `## main / unreleased` items under this release’s version.
 
    > **Note:** If there are any deprecated features that should be removed in this release, then verify that they have been removed, and move their deprecation notices into the section for this release.
 
-   b. Have the PR reviewed by a maintainer.
+   - Have the PR reviewed by a maintainer.
+
+   - Merge the PR upon approval.
+
+1. Create a release branch.
+
+   - Create (if the branch is not created yet), switch to and push a branch starting from the commit created by the prepare changelog PR and name it `mimir-distributed-release-x.y`.
+
+     For example, `mimir-distributed-release-4.5` for any `4.5.x` release.
+
+   - Push the branch to origin without any commit added.
+
+   - Once the branch is pushed, all changes to `mimir-distributed-release-x.y` branch must be done through PR.
+
+1. Create a branch from release branch if it hasn't been created yet, to update Mimir/GEM image and helm chart version .
+
+   For example `user/update-mimir-distributed-release-x.y`.
+
+1. Update versions and links in the `user/update-mimir-distributed-release-x.y` branch.
+
+   - Set the image versions in [values.yaml](https://github.com/grafana/mimir/blob/main/operations/helm/charts/mimir-distributed/values.yaml), as needed:
+
+     - `image.tag` (Mimir)
+     - `enterprise.image.tag` (GEM)
+
+       > **Note:** Unlike the Mimir image tags, GEM image tags start with `v`. For example, `v2.6.0` instead of `2.6.0`.
+
+     - `smoke_test.image.tag` (Smoke test; usually the same as Mimir)
+     - `continuous_test.image.tag` (Continuous test; usually the same as Mimir)
+
+   - Set the `version` field, in the [Chart.yaml](https://github.com/grafana/mimir/blob/main/operations/helm/charts/mimir-distributed/Chart.yaml) file, to the desired release candidate version.
```

Comment:
Let's extract this in a new step (will be step 6). So the preparations can be done without risk of releasing anything.
Let's call the step "Trigger the release"

Reply:
we agreed to keep it here